### DISCUSSION TITLE
Keyboard shortcut to pause all streams at once

### DISCUSSION THREAD

**Discussion by RAS915** [f1multiviewer/issue-tracker#36] [Feature requests] [answered]
It would be good to have some keyboard shortcut (for example Shift + Space) that pause all streams at once. Currently if you stop one stream, the others continue.

Primarily this shortcut should be helpful for replays, but it could also be useful during a live stream.

**Comment by AtlAntA118** [comment]
Yes i would like that! My bladder was about to explode after the last GP!

**Comment by tonogdlp** [comment]
I would add that it can be a good idea to just add `shift` to the existing commands to make it work on all streams. That way:

`Shift` + `spacebar` --> All streams pause/play
`Shift` + `t`               --> All OBC Streams toggle telemetry overlay
`Shift` + `d`              --> All streams toggle debug data overlay
`Shift` + `m`             --> All streams toggle mute.. 

etc..

**Reply by f1multiviewer** [reply]
I think this is a fun one, but best to create a separate discussion for these (no need to create one for each, but a single for "multi-stream control hotkeys" would be fine)

**Comment by JJWatMyself** [comment, marked as answer]
Shift + Space is now implemented, or at least is the case for replays.

I agree that the other Shift + shortcuts would be beneficial.

**Comment by beerse** [comment]
My logic mind says: If the buttons work to a single stream if that is the active window. At the moment the central window is the active window, then all streams should be adressed for the keys. I might make this a differen/new request if it is not adressed already.
